mathics/builtin/__init__.py

```diff
@@ -262,16 +248,15 @@ def sanity_check(cls, module):
                     builtin_class, module
                 ), f"In {module.__name__} Builtin <<{builtin_class.__name__}>> did not pass the sanity check."
 
-                _builtins.append((instance.get_name(), instance))
+                _builtins_list.append((instance.get_name(), instance))
                 builtins_by_module[module.__name__].append(instance)
 
 mathics_to_sympy = {}  # here we have: name -> sympy object
-mathics_to_python = {}  # here we have: name -> string
 sympy_to_mathics = {}
 
 builtins_precedence = {}
 
-new_builtins = _builtins
+new_builtins = _builtins_list
 
 # FIXME: some magic is going on here..
```

Comment:
I now understand a little that what was going on, as described above, was that there are global variables with changing meaning, that type shift and now it is getting  reset here for possibly for some different purpose.